tests/lib/rules/object-curly-newline.js

```diff
@@ -586,6 +586,38 @@ ruleTester.run("object-curly-newline", rule, {
         },
         {
             code: [
+                "var a = {",
+                " /* comment */ ",
+                "};"
+            ].join("\n"),
+            output: [
+                "var a = {",
```

Comment:
Nitpick: Can you use `output: null` for these test cases to indicate that no output is produced, rather than making `output` the same as `code`?